src/sarif_converter.ts

```diff
@@ -187,3 +203,39 @@ const getResults = (suggestions: ISarifSuggestions) => {
   }
   return output;
 };
+
+//custom string splice implementation
+function spliceSlice(str: string, index: number, count: number, add: string) {
+  // We cannot pass negative indexes directly to the 2nd slicing operation.
+  if (index < 0) {
+    index = str.length + index;
+    if (index < 0) {
+      index = 0;
+    }
+  }
+
+  return str.slice(0, index) + (add || '') + str.slice(index + count);
+}
+
+const getArgumentsAndMessage2 = (
```

Comment:
`required`: Where is `getArgumentsAndMessage1`? :-) Please, change name to something more meaningful. Also, please prefer `function getArgumentsAndMessage2() {}` syntax over variable storing.

Reply:
Thank you re the name I completely agree! re the function: Why is this best practise?

Reply:
`const getArgumentsAndMessage2 = ()` will end up as an anonymous function in a stack trace (because it is anonymous, just stored in named variable), whilst `function getArgumentsAndMessage2() {}` is named function, so it will appear in stack trace that way.

This is also valid for another features working with function name.

Reply:
👍  thank you